### DISCUSSION TITLE
publication.status dodatkowy status

### DISCUSSION THREAD

**Discussion by nowyyy31** [allegro/allegro-api#8541] [Suggestion - verification]
Dzień dobry, 

Proszę o dodanie nowego statusu dla aukcji które są zarchiwizowane oraz usunięte ponieważ łamały regulamin. 
Aktualnie takie oferty mają status ENDED i ciężko jest stwierdzić czy można taką aukcję aktywować czy nie. 

Pozdrawiam
Karol

**Comment by Lukasz-Zurek** [comment]
@nowyyy31 Twoją sugestię przenoszę do dedykowanego miejsca [Github Discussions](https://github.com/allegro/allegro-api/discussions).
